client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.jsx

```diff
@@ -120,6 +125,22 @@ const DataFilesBreadcrumbs = ({
   const paths = [];
   const pathComps = [];
 
+  const dispatch = useDispatch();
+
+  const fileData = {
+    system: system,
+    path: path,
+  };
+
+  const openFullPathModal = (e) => {
+    e.stopPropagation();
+    e.preventDefault();
+    dispatch({
+      type: 'DATA_FILES_TOGGLE_MODAL',
+      payload: { operation: 'showpath', props: { file: fileData } },
+    });
+  };
```

Comment:
This appears to not be working within an applications file browser modal
<img width="306" alt="Screenshot 2023-10-18 at 12 04 27 PM" src="https://github.com/TACC/Core-Portal/assets/20326896/303f196a-0fa2-4c27-8713-23db098775ef">

Reply:
Currently investigating this. There's two issues I've identified. In the context of DataFilesMoveModal and DataFilesCopyModal (where the View Full Path button is working) we have file and definition defined in that context. Whereas, in the DataFilesSelectModal, definition is not being properly defined. It's returning 'undefined' in that context. In order for the modal to show up, I have to force re-render the ShowPathModal inside the SelectModal. When I do that it works correctly, however if I refresh the page--the definition value goes back to 'undefined'. and the modal won't open as a result when clicking the View Full Path button. So long story short, I need to figure out how the states are being handled and changed in the SelectModal context, and figure out why DataFilesShowPathModal isn't re-rendered/mounted in the SelectModal context.

Reply:
Fixed